diesel_cli/src/main.rs

```diff
@@ -460,6 +460,14 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<dyn Error>> {
         config.with_docs = true;
     }
 
+    if let Some(sorting) = matches.value_of("column-sorting") {
+        match sorting {
+            "ordinal_position" => config.column_sorting = ColumnSorting::OrdinalPosition,
+            "name" => config.column_sorting = ColumnSorting::Name,
+            _ => unreachable!("Invalid column sorting mode: {}", sorting),
```

Comment:
Can we return an `Err` here instead of panicing?

Reply:
I've just fixed this!